moped-editor/src/utils/activityLogFormatters/mopedFundingActivity.js

```diff
@@ -10,46 +10,83 @@ export const formatFundingActivity = (
   const entryMap = ProjectActivityLogTableMaps["moped_proj_funding"];
 
   const changeIcon = <MonetizationOnOutlinedIcon />;
-  let changeDescription = "Project funding updated";
-  let changeValue = "";
 
   // add a new funding source
   if (change.description.length === 0) {
-    changeDescription = "Added a new funding source";
     // if the added record has a funding source, use that as the change value
     if (change.record_data.event.data.new.funding_source_id) {
-      changeValue =
-        fundingSources[change.record_data.event.data.new.funding_source_id];
+      return {
+        changeIcon,
+        changeText: [
+          { text: "Added ", style: null },
```

Comment:
related to my comment about only adding a className when rendering if this style field exists, would it be worth leaving this key value pair out of the object if there is no necessary styling? just a thought!

Reply:
I did this so that changeObject.style would be null and not undefined, but I didn't double check to confirm thats what was happening